test/e2e/storage/volume_metrics.go

```diff
@@ -95,6 +100,53 @@ var _ = SIGDescribe("[Serial] Volume metrics", func() {
 			verifyMetricCount(storageOpMetrics, updatedStorageMetrics, volumeOp)
 		}
 	})
+
+	It("should create prometheus metrics for volume summary stats", func() {
```

Comment:
refactor the message to be more reflective of the actual test running

Reply:
👍 - will change to `"should create volume metrics with the correct PVC ref"`

Reply:
I don't see this change yet.